public/js/monitor.js

```diff
@@ -145,6 +145,10 @@ function openModalWindow() {
   ga_sendPing("SignUp");
   const subscribeModal = document.getElementById("subscribe-modal");
   document.body.classList.add("show-subscribe-modal");
+  const currentEmail = sessionStorage.getItem("current_email");
```

Comment:
Does sessionstorage throw any error if it, or cookies are disabled?

Reply:
No errors thrown without sessionStorage. I'm going to merge for a release today.